sympy/stats/tests/test_stochastic_process.py

```diff
@@ -643,6 +643,9 @@ def test_WienerProcess():
     Contains(d, Interval.Ropen(1, 2)) & Contains(t, Interval.Lopen(0, 1)))
     assert E(X(t) + x*E(X(3))) == 0
 
+    # test issue 20078
+    assert str((2*X(t)).simplify()) == '2*X(t)'
```

Comment:
String comparisons should be avoided.

Reply:
Add more tests like, `simplify(X(t) + X(t)) == 2*X(t)` to verify that the simplification actually works.